env.source.sample

```diff
@@ -37,6 +37,18 @@ token \
 here \
 "
 
+### OPS_UTILS_DIR
+### The OPS_UTILS_DIR setting is an absolute path to any necessary scripts you wish
+### to have automatically mounted into your container. This is mounted in the
+### `/root/sop-utils` directory in the container, and you can optionally add that to
+### the path below if you wish
+# export OPS_UTILS_DIR=/Users/myuser/path/to/ops-sop/v4/utils
+
+### OPS_UTILS_DIR_RW is a boolean flag to allow mounting the ops-sop directory as
+### read/write instead of the default readonly. This is useful when debugging or 
+### writing a script.
+# export OPS_UTILS_DIR_RW=true
+
```

Comment:
all of this is bash code, so you can just change that on your machine

Reply:
I'm not sure what you mean.  Yes, it's bash, but I think having this individually configurable rather than hard-coded means that it's easier to turn on/off rather than re-writing the bash script that then makes you out of sync with the main branch in case you try to update later.

Reply:
SGTM